charms/tensorboards-web-app/tests/unit/test_operator.py

```diff
@@ -0,0 +1,115 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+from unittest.mock import MagicMock, patch
+
+import ops.testing
+import pytest
+import yaml
+from ops.model import ActiveStatus, BlockedStatus, MaintenanceStatus, WaitingStatus
+from ops.testing import Harness
+
+from charm import TensorboardsWebApp
+
+# Enable simulation of container networking
+ops.testing.SIMULATE_CAN_CONNECT = True
```

Comment:
Since we're moving to `ops 2.3.0`, this is enabled by default, so explicitly setting it here is not required. See here for more: https://juju.is/docs/sdk/write-a-unit-test-for-a-charm#heading--simulate-can-connect